corehq/apps/app_manager/templates/app_manager/spec/mocha.html

```diff
@@ -3,8 +3,14 @@
 
 {% block dependencies %}
 <script src="{% static 'app_manager/js/form_workflow.js' %}"></script>
+<script src="{% static 'app_manager/js/download_async_modal.js' %}"></script>
 {% endblock %}
 
 {% block mocha_tests %}
 <script src="{% static 'app_manager/spec/form_workflow_spec.js' %}"></script>
+<script src="{% static 'app_manager/spec/download_async_modal_spec.js' %}"></script>
```

Comment:
ah, i think it's more common in the ruby world. `rspec` is a very popular tool: http://rspec.info/ and jasmine also uses `spec`